### PR TITLE
support DeviceSupply

### PR DESCRIPTION
**Pull request by eguchi1904**
## Check list

- [ ] I wrote [CHANGELOG.md](./CHANGELOG.md) if the pull request adds/modifies features.

### CODE REVIEW

**Comment by eguchi1904** [comment]
confirmed  `get_device_supply` behavior in the latest actsim enviroment.

- manifesto
```json
{
  "version": 2,
   "target_type": "raspberrypi-bullseye",
  "boards": [
    "RSPi4B"
  ],
  "devices": [
    {
      "device": [
        "/dev/video0"
      ],
      "type": "camera"
    },
    {
      "type": "videocore"
    },
    {
      "type": "display"
    }

  ]
}

```

- app
```python
from actfw_core.system import get_device_supply
...
    print(get_device_supply())

```

- output
```
DeviceSupply(devices=[DeviceInfo(type='camera', nodes=[DeviceNode(path=PosixPath('/dev/video0'), label=None)]), DeviceInfo(type='videocore', nodes=[DeviceNode(path=PosixPath('/dev/vcsm-cma'), label=None), DeviceNode(path=PosixPath('/dev/vcio'), label=None), DeviceNode(path=PosixPath('/dev/vc-mem'), label=None), DeviceNode(path=PosixPath('/dev/vchiq'), label=None), DeviceNode(path=PosixPath('/dev/gpiomem'), label=None), DeviceNode(path=PosixPath('/dev/dri/card0'), label=None), DeviceNode(path=PosixPath('/dev/dri/card1'), label=None)]), DeviceInfo(type='display', nodes=[DeviceNode(path=PosixPath('/dev/dri/card0'), label=None), DeviceNode(path=PosixPath('/dev/dri/card1'), label=None)])])
```

File: actfw_core/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import Any, Callable, Dict, List
 
-from . import capture, linux, task, unicam_isp_capture  # noqa: F401
+from . import capture, linux, system, task, unicam_isp_capture  # noqa: F401
```
Comment (ishiy1993) [inline actfw_core/__init__.py:5]: conflict with https://github.com/Idein/actfw-core/pull/81.

@KeenS @eguchi1904 Which is it better to close #81 or delete this commit?

Comment (KeenS) [reply actfw_core/__init__.py:5]: Sorry, I've already merged #81 . I guess `git merge` so clever that they won't conflict.

**Review by KeenS — APPROVED** [review]

**Review by ishiy1993 — APPROVED** [review]

**Comment by eguchi1904** [comment]
:pray:

**Pull request merged**